Gpu lab edits (#53)
CHTC/chtc-website-source: gpu-jobs.md

## A. General Use GPUs via the CHTC GPU Lab

CHTC has a set of GPUs that are available for use by any CHTC user with an 

> account on our high throughput computing (HTC) system. These shared use 

Reviewer: ```suggestion
account on our high throughput computing (HTC) system
via the [CHTC GPU Lab](/gpu-lab.shtml), which includes templates and a campus GPU community.
```

GPUs form the [CHTC GPU Lab](/gpu-lab.shtml). 

> Because the CHTC GPU Lab is limited in size, and because the majority of 

Reviewer: ```suggestion
Because the CHTC GPU Lab was
funded by a special investment from the [UW2020 program](https://research.wisc.edu/funding/uw2020/round-5-projects/enabling-graphics-processing-unit-based-data-science/), we
```

its GPUs were funded by a special investment from the UW2020 program, we 
have implemented special policies to maximize how many researchers can 
benefit from this investment. These policies are described briefly below, 
and in greater length in the section of this guide that describes the 
needed [submit file modifications to use the CHTC GPU Lab](#b-use-the-gpu-lab-servers-recommended).

> **Jobs running in the CHTC GPU Lab have different time and job number limits than normal CHTC job

Reviewer: ```suggestion
**Jobs running on GPU Lab servers have time limits and job number limits (differing from CHTC defaults across the rest of the HTC System).
```

> submissions**. Without any special requirements, jobs that use the CHTC GPU Lab 

Reviewer: ```suggestion
Without any special requirements, jobs that use the CHTC GPU Lab 
```

> directly can run for up to 24 hours and a single user will be able to use up to 1/3 of 

Reviewer: Maybe skip having prose, and just let this info be in the table?

Author: I'll make that change in my next update.

## A. General Use GPUs via the CHTC GPU Lab

CHTC has a set of GPUs that are available for use by any CHTC user with an 
account on our high throughput computing (HTC) system
via the [CHTC GPU Lab](/gpu-lab.shtml), which includes templates and a campus GPU community.

> GPUs form the [CHTC GPU Lab](/gpu-lab.shtml). 

Author: ```suggestion

```

  </tr>
  <tr>
    <td>4</td>
    <td>gpulab2000 - gpulab2003</td>
    <td>8</td>
    <td>GeForce RTX 2080 Ti</td>
    <td>CentOS 7</td>
    <td>yes</td>
  </tr>
</table>

By opting-in to use the CHTC GPU Lab servers, you agree to be contacted by the
project leaders occasionally to discuss your GPU computing and help improve the
GPU Lab.

> ## B. Researcher Owned GPUs

Reviewer: Is it a problem to have two "B" items?

Reviewer: Maybe put "Researcher-Owned GPUs" (note the hyphen) lower, making sure to link from the main menu/linked list of topics on this page?

Author: If I put it lower, I feel like I'd be breaking up the page. Right now it's: 
- resources
- submit file
- software

Author: I could restructure to be: 
- shared use (GPU Lab)
- non-shared use
- software

The tricky piece there, is that some things apply across the board -- like, everybody needs to request GPUs. 

Maybe I'll try to draft that up and submit as a separate PR for you to look at.

shouldn't need to set any environmental variables or other options
related to the GPU, except what is needed inside your code.

It is possible to request multiple GPUs. Before doing so, make sure you're 

> using code that can utilize multiple GPUs and then submit a test to confirm 

Reviewer: ```suggestion
using code that can utilize multiple GPUs and then submit a test job to confirm 
```

> this works before submitting a bigger job. Also keep track of how long jobs 

Reviewer: ```suggestion
success before submitting a bigger job. Also keep track of how long jobs 
```

Reviewer: avoid "this" as a noun, as it's vague.

are running versus waiting; the time you save by using multiple GPUs may be 
not worth the extra time that the job will likely wait in the queue. 

## B. Use the GPU Lab Servers (recommended)

> To use CHTC's shared use GPUs, you need to opt-in to the GPU Lab. To 

Reviewer: Maybe figure out how to have this show up sooner?

Author: Same comments as previously -- would require a restructuring of the guide.
